content/crutchcorn/collections/art-of-accessibility/posts/art-of-a11y-text/index.md

```diff
@@ -0,0 +1,561 @@
+
+```html
+<!-- This is bad -->
+<!-- "Collections" and "Posts" shoud be h2s -->
+<h1>Search page</h1>
+<!-- ... -->
+<h1>Collections</h1>
+<!-- ... -->
+<h1>Posts</h1>
+<!-- ... -->
+```
+
+### Descending heading orders
+
+If we asked our imaginary population to count to `10` again, we'd find that most folks are likely to assume an increment of one counting up: `1`, `2`, `3`, and so on.
+
+After all, it would be atypical for someone to go from `2` to `4` when counting upwards, right?
+
+The same applies for headings: **Headings are expected to be added in a descending order of `1` level of depth at a time**.
+
+For example:
+
+```html
+<h2>Heading two</h2>
+<!-- Good! Went from 2 to 3 -->
+<h3>Heading three</h3>
+<!-- Bad! Missing an `h4` -->
+<h5>Heading three</h5>
+```
+
+However, while you should only count heading levels down by `1` at a time, you can jump back up the count by any number you'd like:
+
+``` html
+<!-- This is all allowed -->
+<h2>Heading two</h2>
+<h3>Heading three</h3>
+<h4>Heading four</h4>
+<h2>Heading five</h2>
+```
+
+## Content Grouping
+
+When working with content that has a distinct grouping of information, it may be tempting to assume that the user can simply use headings to distinguish different parts of the site.
+
+Alas, this isn't true! Take the following example:
+
+![A headphone sales marketing page with a description, features list, and customer reviews](./headphones_mockup.png)
+
+This mockup might be naïvely marked up as such:
+
+```html
+<h1>EchoFlow X1</h1>
+<img src="headphone.jpg" alt="Black headphones with 'EchoFlow X' written on the earcup.">
+<h2 class="visually-hidden">Description</h2>
+<p>The EchoFlow X1 is the world's first...</p>
+<h2>Features</h2>
+<ul>
+    <li>Infinite Playback</li>
+    <li>Adaptive Sound</li>
+</ul>
+<h2>Customer Reviews</h2>
+<p>Mind blowing! It's like...</p>
+<p>A game-changer for music...</p>
+```
+
+But this introduces a problem: Where does one part of the app stop and another part begin?
+
+This is where `<section>` tags come into play. Let's change our markup to the following:
+
+```html
+<h1>EchoFlow X1</h1>
+<main>
+  <img src="headphone.jpg" alt="Black headphones with 'EchoFlow X' written on the earcup.">
+
+  <section>
+    <h2 class="visually-hidden">Description</h2>
+    <p>The EchoFlow X1 is the world's first...</p>
+  </section>
+
+  <section>
+    <h2>Features</h2>
+    <ul>
+        <li>Infinite Playback</li>
+        <li>Adaptive Sound</li>
+  	</ul>
+  </section>
+
+  <section>
+    <h2>Customer Reviews</h2>
+    <ul>
+        <li>Mind blowing! It's like...</li>
+        <li>A game-changer for music...</li>
+      </ul>
+  </section>
+</main>
+```
+
+Now it's more clear which headings belong to which sections.
+
+>  **Keep in mind:**
+>
+> Sections should almost always have a heading as the first child; otherwise it's challenging to figure out what a given section is intended to do.
```

Comment:
This is a great rule of thumb for making good documents, but not strictly speaking required or exposed, and potentially misleading. Recommending this already had boxed us into having to use `visually-hidden`.